packages/create-plugin/src/commands/provisioning.command.ts

```diff
@@ -0,0 +1,22 @@
+import glob from 'glob';
+import { TEMPLATE_PATHS, TEXT } from '../constants';
+import { compileProvisioningTemplateFile, getTemplateData } from '../utils/utils.templates';
+import { confirmPrompt, printMessage, printSuccessMessage } from '../utils/utils.console';
+
+export const provisioning = async () => {
+  try {
+    if (await confirmPrompt(TEXT.updateProvisioning)) {
```

Comment:
Would this command overwrite existing provisioning files? If so, can we detect if a provisioning is already present in the plugin and ask accordingly? e.g.:  "You already have provisioning, would you like to replace it?"

Or, going further, if the plugin already has provisioning, is there any value on even asking the user if they want to replace it? should we instead say "You plugin already has provisioning files located in ${path}" ?